Lesson1/Task1/Program.cs

```diff
@@ -4,20 +4,33 @@ namespace Task1
 {
     class Program
     {
+        static double Check()
+        {
+            double checkedInput = 0;
+            string input = Console.ReadLine();
+            while (!double.TryParse(input, out checkedInput))
+            {
+                Console.Write("Integer required! Try again: ");

```

Comment:
@okuzmina You are trying to parse from double, but on the console user will read "Integer", it's strange, should be changed